Remotion/Web/UnitTests/Core/Security/UI/WebSecurityAdapterTests/WebPermissionProviderTestHelper.cs

```diff
@@ -30,97 +30,94 @@ public class WebPermissionProviderTestHelper
-    private readonly IPrincipalProvider _mockPrincipalProvider;
-    private readonly IObjectSecurityStrategy _mockObjectSecurityStrategy;
-    private readonly IFunctionalSecurityStrategy _mockFunctionalSecurityStrategy;
-    private readonly IWxeSecurityAdapter _mockWxeSecurityAdapter;
+    private readonly Mock<ISecurityPrincipal> _stubUser;
+    private readonly Mock<ISecurityProvider> _mockSecurityProvider;
+    private readonly Mock<IPrincipalProvider> _mockPrincipalProvider;
+    private readonly Mock<IObjectSecurityStrategy> _mockObjectSecurityStrategy;
+    private readonly Mock<IFunctionalSecurityStrategy> _mockFunctionalSecurityStrategy;
+    private readonly Mock<IWxeSecurityAdapter> _mockWxeSecurityAdapter;
 
     // construction and disposing
 
     public WebPermissionProviderTestHelper ()
     {
-      _mocks = new MockRepository();
-
-      _mockSecurityProvider = _mocks.StrictMock<ISecurityProvider>();
-      SetupResult.For (_mockSecurityProvider.IsNull).Return (false);
-      _mockObjectSecurityStrategy = _mocks.StrictMock<IObjectSecurityStrategy>();
-      _mockFunctionalSecurityStrategy = _mocks.StrictMock<IFunctionalSecurityStrategy>();
-      _mockWxeSecurityAdapter = _mocks.StrictMock<IWxeSecurityAdapter>();
-
-      _stubUser = _mocks.Stub<ISecurityPrincipal>();
-      SetupResult.For (_stubUser.User).Return ("user");
-      _mockPrincipalProvider = _mocks.StrictMock<IPrincipalProvider>();
-      SetupResult.For (_mockPrincipalProvider.GetPrincipal()).Return (_stubUser);
+      _mockSecurityProvider = new Mock<ISecurityProvider> (MockBehavior.Strict);
+      _mockSecurityProvider.Setup (_ => _.IsNull).Returns (false);
+      _mockObjectSecurityStrategy = new Mock<IObjectSecurityStrategy> (MockBehavior.Strict);
+      _mockFunctionalSecurityStrategy = new Mock<IFunctionalSecurityStrategy> (MockBehavior.Strict);
+      _mockWxeSecurityAdapter = new Mock<IWxeSecurityAdapter> (MockBehavior.Strict);
+
+      _stubUser = new Mock<ISecurityPrincipal>();
+      _stubUser.Setup (_ => _.User).Returns ("user");
+      _mockPrincipalProvider = new Mock<IPrincipalProvider> (MockBehavior.Strict);
+      _mockPrincipalProvider.Setup (_ => _.GetPrincipal()).Returns (_stubUser.Object);
     }
 
     // methods and properties
 
     public void ExpectHasAccess (Enum[] accessTypeEnums, bool returnValue)
     {
       AccessType[] accessTypes = Array.ConvertAll<Enum, AccessType> (accessTypeEnums, AccessType.Get);
-      Expect.Call (
-          _mockObjectSecurityStrategy.HasAccess (
-              Arg.Is (_mockSecurityProvider),
-              Arg.Is (_stubUser),
-              Arg<IReadOnlyList<AccessType>>.List.Equal (accessTypes)))
-          .Return (returnValue);
+      _mockObjectSecurityStrategy.Setup (
+              _ => _.HasAccess (
+                  _mockSecurityProvider.Object,
+                  _stubUser.Object,
+                  accessTypes))
+          .Returns (returnValue)
+          .Verifiable();
```

Comment:
```suggestion
      _mockObjectSecurityStrategy
          .Setup (
              _ => _.HasAccess (
                  _mockSecurityProvider.Object,
                  _stubUser.Object,
                  accessTypes))
          .Returns (returnValue)
          .Verifiable();
```
(since I'm already touching this and it's right next to the other one...)